webclient/src/views/NotFoundView.vue

```diff
@@ -2,14 +2,15 @@
 import { useGlobalStore } from "@/stores/global";
 
 const global = useGlobalStore();
+const href = window.location.href;
 </script>
 
 <template>
   <div class="toast toast-error">{{ $t("core_js.error.404") }}</div>
   <h5 style="margin-top: 25px">{{ $t("view_404.header") }}</h5>
   <p>{{ $t("view_404.description") }}</p>
   <button
-    @click="global.openFeedback('bug', `404 on ${window.location.href}`, $t('view_404.got_here'))"
+    @click="global.openFeedback('bug', `404 on ${href}`, $t('view_404.got_here'))"
```

Comment:
```suggestion
    @click="global.openFeedback('bug', `404 on ${window.location.href}`, $t('view_404.got_here'))"
```
I think this could point to the wrong site (`script setup` is only executed once)